wordpress-mobile/WordPress-Android
# Issue/backup restore reduce fetch delay

Parent #13328 & #13329

Merge Instructions:
1. Make sure [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1860) is merged first
2. Update FluxC tag in gradle file
3. Remove "Not ready for merge label"

To test:
1. Launch app
2. Turn on the Restore feature flag
3. Navigation to Activity Log
4. Tap on the more menu on a row 
5. Select the "Restore to this point' option
6. Follow the restore process to end
7. Note that the process runs as expected

------------
1. Launch app
2. Turn on BackupDownload feature flag
3. Navigation to Activity Log
4. Tap on the more menu on a row 
5. Select the "Download backup' option
6. Follow the download process to end
7. Note that the process runs as expected

PR submission checklist:
